.github/workflows/build.yml

```diff
@@ -102,4 +216,71 @@ jobs:
       - name: Run tests
         env:
           RELEASE_TAG: ${{ github.event.release.tag_name }}
-        run: pytest
+        run: pytest --runslow
+  build-push-all:
+    name: "Build and push all platforms"
+    runs-on: ubuntu-latest
+    needs: [prepare, test]
+    if: github.event_name != 'pull_request'
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.BUILDX_CACHE_DIR }}
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
```

Comment:
We've been creating keys of the form `buildx-${{ runner.os }}...` for a while now.  This is a change that @mcdonnnj encouraged some time ago.

Reply:
Ah ha.  I found the reason I "reverted" to the ways of old.  It was good ol' cribbin' on my part from the Docker folks: 
https://github.com/docker/build-push-action#leverage-github-cache

Reply:
I assumed that was the case.  The Great and Powerful Oz knows all!

Reply:
I thought it might've been a copypasta incident. That's why we review though!